snf-cyclades-app/synnefo/userdata/models.py

```diff
@@ -57,9 +58,23 @@ class PublicKeyPair(ProfileModel):
     content = models.TextField(validators=[
         MaxLengthValidator(SSH_KEY_MAX_CONTENT_LENGTH)])
     fingerprint = models.CharField(max_length=100, null=False, blank=True)
+    created_at = models.DateTimeField(auto_now_add=True,
```

Comment:
What I suggested was to have a sensible default value for pre-existing entities. What @gkorf says is that this default value should be set during migration for pre-existing entities . Perhaps I wasn't clear enough or the use of 'default' keyword was misleading..